src/Operator/Constructors.jl

```diff
@@ -598,7 +699,7 @@ operator field:
     interpolation
 ```
 """
-function massoperator(basis::AbstractBasis, mesh::Mesh)
+function massoperator(basis::AbstractBasis, mesh::Mesh; parameters = nothing)
```

Comment:
Does documentation warn about the missing argument? I feel like we should document that this should be nothing at this time for our own internal notes since this isn't part of the external API

Reply:
Where and how can the warning be found?

Reply:
It would show up when building the documentation, locally or in CI

Reply:
In any case we should probably make sure that parameters is documented in all these functions